AppleMusicClone/Views/SearchViews/SearchView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  SearchView.swift
+//  AppleMusicClone
+//
+//  Created by Hankyu Lee on 2022/07/05.
+//
+
+import SwiftUI
+import Combine
+
+struct SearchView: View {
+    
+    @State private var location = ""
+
+    @StateObject var searchVM: SearchViewModel = SearchViewModel()
```

Comment:
```suggestion
    @StateObject private var searchViewModel: SearchViewModel = SearchViewModel()
```
1. `searchVM` 은 `SearchView` 파일에서만 사용하는 변수로 보여요!
2. `searchVM` 같이 축약형으로 많이 쓰이나요? Swift에서는 축약하지 않는 것을 권장해서 `searchViewModel`이 어떨까 싶은데 자주 쓰이는 축약어라면 좋습니다!

Reply:
## Suggest
저도 공식문서에 줄여쓰는거 안좋아하는거 같아서 ViewModel이 좋은거같습니다!

<img width="1154" alt="image" src="https://user-images.githubusercontent.com/103009135/177726139-26ad2b61-0688-4728-9bc6-ff1f14d2e54c.png">

Reply:
감사합니다 수정하였습니다!